code/modules/paperwork/photography.dm

```diff
@@ -464,8 +460,13 @@
 		var/wanted = copytext(sanitize(input(user, "Who would you like to take a photograph of?", "Target name")as text | null),1,26)
 		if(wanted in real_living_players)
 			target = real_living_players[wanted]
+			var/turf/target_turf = get_turf(target)
+			if(target_turf.z == 2)
```

Comment:
https://github.com/HippieStationCode/HippieStation13/blob/5462010e5bcc8a61ad5618ffd0f04f07eeb570b0/code/game/atoms.dm#L22 use this proc instead, it's better